requirements/dev.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
```

Comment:
These dep changes don't look right. Can you regenerate on Python 3.8?

Reply:
yeah, I can do that.